google/cloud/pubsub/message.h

```diff
@@ -294,6 +300,15 @@ inline void SetAttribute(std::string const& key, std::string value,
   (*m.proto_.mutable_attributes())[key] = std::move(value);
 }
 
+inline std::string GetAttributeValue(std::string const& key,
```

Comment:
I would eschew `inline` unless it is deemed a performance hot spot.

Reply:
This is in a header, I think it needs to be inline or moved to the cc file.

Reply:
Right you are ... performance has nothing to do with it (unless we're talking about compiler/linker performance).  What I'm really trying to suggest is moving things to the cc file when you can (particularly when one already exists).

Reply:
Moved all functions to the cc